js/ControlTodoItem.js

```diff
@@ -0,0 +1,28 @@
+const $todoList = document.getElementById("todo-list");
+
+export function initControlTodoItem(){
+    $todoList.addEventListener("click", onToggleTodoItem);
+}
+
+//완료한 일로 check, 할 일 삭제 : edit모드에서는 작동X
+function onToggleTodoItem(e) {
+    const li = e.target.closest("li");
+
+    //할 일을 체크하거나 해제
+    if(li.classList.value === "" || li.classList.value === "completed"){
+        //완료된 일 
+        if(e.target && e.target.nodeName === 'INPUT'){ 
+            e.target.closest("li").classList.toggle("completed");
```

Comment:
```suggestion
            li.classList.toggle("completed");
```
위에서 정의된 `li`와 같은 객체네요 ☺️

Reply:
미쳐 수정하지 못한 부분이네요. 😢 
마저 수정하여 review#12에 반영하였습니다. 
좋은 피드백 감사합니다. 😄